mysteryword.py

```diff
@@ -0,0 +1,59 @@
+#$open a file on the server - in our example "words.txt"
+# with open("words.txt", "r") as file:
+#     wordz = file.read().splitlines()
+# print(wordz)
+
+# for version 2.0!!!!!
+# select a random word (line) on the server 
+#$if input is more than 2 character limit only respond with Nope try again
+
+# this will come for version 2.0!!!!!!
+# user choose level of difficulty
+# easy 4-6 characters if/then statments
+# normal 6-8 characters if/then statements
+# hard 8+ characters if/then statments
+
+#$select word to get started = "pizza" for the test 
+word = "pizza"
+print(len(word))
+
+#show in console the number of letters in computers' word 
+
+#user enter guess - one letter only (uppercase/lowercase), at a time per round
+#set limit to input to 1 character
+#or else tell user input invalid
+
+def playgame():
+    guess = input("Choose a letter ")
```

Comment:
You want some kind of loop that keeps the guess process going until it's time to stop. Could be inside this function or outside of the function and calls `playgame()`